spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtils.java

```diff
@@ -45,4 +45,8 @@ public static String getApplicationNamespace(KubernetesClient client, String con
 		return configNamespace;
 	}
 
+	static String getNamespace(KubernetesClient client, String namespace) {
```

Comment:
- has moved from `Fabric8ConfigMapPropertySource` in here. Seems to me a more appropriate place to be. Both classes are in the same package, and to preserve the `private` modifier that was present before, we keep it here without any modifiers at all. So this is still "hidden" from the external world, just that now it is scoped at package level. 
- I have also removed the deprecated `StringUtils::isEmpty` with the inverted `StringUtils::hasLength`